src/sfizz/Wavetables.cpp

```diff
@@ -389,7 +389,7 @@ const WavetableMulti* WavetablePool::createFileWave(FilePool& filePool, const st
     };
 
     auto wave = std::make_shared<WavetableMulti>(
-        WavetableMulti::createForHarmonicProfile(hp, 1.0));
+        WavetableMulti::createForHarmonicProfile(hp, 1.0, config::tableSize, fileHandle->information.sampleRate));
```

Comment:
It's intentional that sample rate is not passed here.
A default 44100Hz is always fine, there is never need of harmonics > 22 kHz.

Beside all synth instances might share wavetables at some point; 44.1k wavetables may be used in a 96k synth for instance, but the reverse will not be true.